lib/handler/configurator/compress.c

```diff
@@ -155,6 +161,9 @@ void h2o_compress_register_configurator(h2o_globalconf_t *conf)
     c->super.enter = on_config_enter;
     c->super.exit = on_config_exit;
     h2o_configurator_define_command(&c->super, "compress", H2O_CONFIGURATOR_FLAG_ALL_LEVELS, on_config_compress);
+    h2o_configurator_define_command(&c->super, "compress-min-size",
```

Comment:
Would you mind changing the name of the configuration directive (as well as the functions) from using `min` to `minimum`?

I ask this because we already have a property named `minimum-version` in the SSL configurator, and I am afraid people might get confused between the use of _min_ and _minimum_.